<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewerClass</class>
 <widget class="QMainWindow" name="ImageViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBoxSetUp">
      <property name="title">
       <string>Set Up</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonColorPalette">
         <property name="text">
          <string>Color Palette</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonDraw">
         <property name="text">
          <string>Activate Drawing</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBoxTransformations">
         <property name="title">
          <string>Transformations</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="8" column="1">
           <widget class="QPushButton" name="pushButtonSymetry">
            <property name="text">
             <string>Symetry</string>
            </property>
           </widget>
          </item>
          <item row="7" column="4">
           <widget class="QDoubleSpinBox" name="spinBoxShear">
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="4">
           <widget class="QDoubleSpinBox" name="spinBoxScale_1">
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="3">
           <widget class="QComboBox" name="comboBoxInterpolation">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Nearest Neighbor</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Barycentrick</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>(deg)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QPushButton" name="pushButtonPolygone">
            <property name="text">
             <string>Polygone</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QDoubleSpinBox" name="spinBoxRotate">
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Algorithm</string>
            </property>
           </widget>
          </item>
          <item row="7" column="3">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>factor</string>
            </property>
           </widget>
          </item>
          <item row="6" column="4">
           <widget class="QDoubleSpinBox" name="spinBoxScale_2">
            <property name="minimum">
             <double>-99.989999999999995</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pushButtonScale">
            <property name="text">
             <string>Scale</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="pushButtonShear">
            <property name="text">
             <string>Shear</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="pushButtonRotate">
            <property name="text">
             <string>Rotate</string>
            </property>
           </widget>
          </item>
          <item row="6" column="3">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>factor (y)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>factor (x)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Interpolacia</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBoxFill">
            <property name="text">
             <string>Fill Polygone/Triangle</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QComboBox" name="comboBoxAlg">
            <item>
             <property name="text">
              <string>DDA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bresenhamov</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxCurves">
      <property name="title">
       <string>Curves</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButtonBezier">
         <property name="text">
          <string>Bezier Curve</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButtonCoons">
         <property name="text">
          <string>Coons Curve</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonHermite">
         <property name="text">
          <string>Hertmite Curve</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButtonClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionRename"/>
    <addaction name="actionClear"/>
    <addaction name="actionSet_background_color"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImage"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSet_background_color">
   <property name="text">
    <string>Set background color</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ImageViewer.qrc"/>
 </resources>
 <connections/>
</ui>
